scripts/superdesk-authoring/services.js

```diff
@@ -1,7 +1,62 @@
 (function() {
 'use strict';
 
-angular.module('superdesk.authoring').service('renditions', RenditionsService);
+angular.module('superdesk.authoring')
+    .service('renditions', RenditionsService)
+    .factory('history', HistoryFactory);
+
+/**
+ * Watches an expression to keep history of its states
+ * and binds ctrl-z and ctrl-y to undo/redo its states
+ */
+HistoryFactory.$inject = ['History', '$window', '$timeout'];
+function HistoryFactory(History, $window, $timeout) {
+    var KeyOperations = {};
```

Comment:
This isn't necessary, but I think it will work now with ES6:
```
var KeyOperations = {
    'Z'.charCodeAt(0): History.undo,
    'Y'.charCodeAt(0): History.redo
};
```